exporter/awsemfexporter/README.md

```diff
@@ -0,0 +1,37 @@
+# AWS CloudWatch EMF Exporter for OpenTelemetry Collector
+
+This exporter converts OpenTelemetry metrics to 
+[AWS CloudWatch Embedded Metric Format(EMF)](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html)
+and then sends them directly to CloudWatch Logs using the 
+[PutLogEvents](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html) API.
```

Comment:
Not really for this PR but is it easy to support both API sending and writing to STDOUT? I guess STDOUT can help in many cases too.